clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -366,6 +384,27 @@ export class CopybookDownloadService implements vscode.Disposable {
     copybookNames: CopybookName[],
     quiet: boolean = true,
   ): Promise<void> {
+    const e4e = await E4ECopybookService.getE4EClient(documentUri);
+    if (
+      e4e &&
+      (e4e.elements.filter((element) =>
+        copybookNames.find((x) => x.name === element.element),
+      ) ||
+        e4e.members.filter((member) =>
+          copybookNames.find((x) => x.name === member.member),
+        ))
```

Comment:
this part will be always true. the condition is equivalent `e4e && ([...] || [...])` so `e4e`